content/geometry/object-human/human-structure.adoc

```diff
@@ -197,7 +197,7 @@ This object represents an additional or exchangeable hair part of the human.
 
 == Root
 
-The `Root` bone is the parent bone for all other bones. It can be used to control the whole skeleton.
+The `<Root>` bone is the parent bone for all other bones. It can be used to control the whole skeleton.
```

Comment:
Why did you add <>? Also for all the other bones.

Reply:
To make it consistent with the vehicle chapter. But I can change it back if it is not correct.

Reply:
But in the vehicles chapter <> should only be used for indices. If it is also used for Groups, we have to correct that.

Reply:
You are right, this is confusing. Currently, groups are also written with code markup in the vehicle chapter if they appear in the description (like here: https://asam-ev.github.io/OpenMATERIAL/asamopenmaterial/latest/specification/geometry/object-vehicle/vehicle-index.html#_grp_mirror_blindspot_mounting_blindspot_mirror_mounting_idx_t), that is where I got the idea from.
But then might it be best to have no mark-up at all for groups in the vehicle chapter and bones in the human chapter and have it only like that (for example): 
- "The Eye_Left bone represents the left eye of the human body. "
- "It is a child node of Grp_Exterior_Dynamic if mounted directly to the vehicle body, or a child node of either Grp_Door_Left or Grp_Door_Right if mounted to the door."

Reply:
Yes, you are right. Let's not use markups for nodes. But I would use them for indices.

Reply:
I implemented this and the other open points.